[2022-08-11 20:55:27,597] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V7.Load_into_monthly_sales_table manual__2022-08-11T02:24:39.409501+00:00 [queued]>
[2022-08-11 20:55:27,608] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: ETL_DAG_V7.Load_into_monthly_sales_table manual__2022-08-11T02:24:39.409501+00:00 [queued]>
[2022-08-11 20:55:27,609] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 20:55:27,610] {taskinstance.py:1377} INFO - Starting attempt 1 of 6
[2022-08-11 20:55:27,611] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-11 20:55:27,623] {taskinstance.py:1397} INFO - Executing <Task(S3ToSnowflakeOperator): Load_into_monthly_sales_table> on 2022-08-11 02:24:39.409501+00:00
[2022-08-11 20:55:27,632] {standard_task_runner.py:52} INFO - Started process 8694 to run task
[2022-08-11 20:55:27,636] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'ETL_DAG_V7', 'Load_into_monthly_sales_table', 'manual__2022-08-11T02:24:39.409501+00:00', '--job-id', '624', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmp4roip_es', '--error-file', '/tmp/tmpvr5hzcov']
[2022-08-11 20:55:27,640] {standard_task_runner.py:80} INFO - Job 624: Subtask Load_into_monthly_sales_table
[2022-08-11 20:55:27,717] {task_command.py:371} INFO - Running <TaskInstance: ETL_DAG_V7.Load_into_monthly_sales_table manual__2022-08-11T02:24:39.409501+00:00 [running]> on host ce1be8c59d56
[2022-08-11 20:55:27,809] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=narapady
AIRFLOW_CTX_DAG_ID=ETL_DAG_V7
AIRFLOW_CTX_TASK_ID=Load_into_monthly_sales_table
AIRFLOW_CTX_EXECUTION_DATE=2022-08-11T02:24:39.409501+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-08-11T02:24:39.409501+00:00
[2022-08-11 20:55:27,811] {s3_to_snowflake.py:135} INFO - Executing COPY command...
[2022-08-11 20:55:27,824] {base.py:68} INFO - Using connection ID 'snowflake_conn' for task execution.
[2022-08-11 20:55:27,828] {connection.py:275} INFO - Snowflake Connector for Python Version: 2.7.11, Python Version: 3.7.13, Platform: Linux-5.10.104-linuxkit-aarch64-with-debian-11.3
[2022-08-11 20:55:27,831] {connection.py:927} INFO - This connection is in OCSP Fail Open Mode. TLS Certificates would be checked for validity and revocation status. Any other Certificate Revocation related exceptions or OCSP Responder failures would be disregarded in favor of connectivity.
[2022-08-11 20:55:27,833] {connection.py:944} INFO - Setting use_openssl_only mode to False
[2022-08-11 20:55:28,731] {cursor.py:715} INFO - query: [ALTER SESSION SET autocommit=True]
[2022-08-11 20:55:28,985] {cursor.py:739} INFO - query execution done
[2022-08-11 20:55:28,986] {snowflake.py:328} INFO - Running statement: COPY INTO public.monthly_sales
FROM @s3_stage/
files=('monthly-sales-staged.csv')
file_format=csv_format, parameters: None
[2022-08-11 20:55:28,987] {cursor.py:715} INFO - query: [COPY INTO public.monthly_sales FROM @s3_stage/ files=('monthly-sales-staged.csv'...]
[2022-08-11 20:55:29,432] {cursor.py:739} INFO - query execution done
[2022-08-11 20:55:29,435] {snowflake.py:338} INFO - Statement execution info - {'file': 's3://s3-bucket-staged/monthly-sales-staged.csv', 'status': 'LOAD_SKIPPED', 'rows_parsed': 0, 'rows_loaded': 0, 'error_limit': None, 'errors_seen': 1, 'first_error': 'File was loaded before.', 'first_error_line': None, 'first_error_character': None, 'first_error_column_name': None}
[2022-08-11 20:55:29,438] {snowflake.py:342} INFO - Rows affected: 1
[2022-08-11 20:55:29,441] {snowflake.py:343} INFO - Snowflake query id: 01a63947-0004-2875-001e-000700035412
[2022-08-11 20:55:29,445] {connection.py:557} INFO - closed
[2022-08-11 20:55:29,590] {connection.py:560} INFO - No async queries seem to be running, deleting session
[2022-08-11 20:55:29,765] {s3_to_snowflake.py:137} INFO - COPY command completed
[2022-08-11 20:55:29,796] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=ETL_DAG_V7, task_id=Load_into_monthly_sales_table, execution_date=20220811T022439, start_date=20220811T205527, end_date=20220811T205529
[2022-08-11 20:55:29,840] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-11 20:55:29,875] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
